### DISCUSSION TITLE
Native Metal API

### DISCUSSION THREAD

**Discussion by alyfreym** [Tencent/tgfx#110] [General] [answered]
Do you planning adding native Metal Api support on IOS? With metal shading language?
Or you planing merge code from Google/ANGLE?

**Comment by domchen** [comment, marked as answer]
We have plans to add support for Metal rendering in the future. For further details, please refer to the "Backing Renderers" section in our [README.md](https://github.com/Tencent/tgfx/blob/main/README.md) file.
